components/Web_driver/Web_driver.c

```diff
@@ -122,16 +127,23 @@ esp_err_t Web_wifi_init(void){
     wifi_init_config_t cfg = WIFI_INIT_CONFIG_DEFAULT();
     ESP_ERROR_CHECK(esp_wifi_init(&cfg));
 
+    ESP_ERROR_CHECK(esp_wifi_get_mac(WIFI_IF_STA, mac));
+    // last two mac address digits should be sufficient to make SSID unique
+    snprintf((char *)mac_addr, sizeof(mac_addr), "%02X%02X", mac[4], mac[5]);
+    ESP_LOGI(TAG, "MAC address: %s", mac_addr);
+
+    memset(ssid, 0x00, ssid_size);
+    snprintf((char *)ssid, ssid_size, "%s-%s", CONFIG_ESP_WIFI_SSID, mac_addr);
+    ESP_LOGI(TAG, "SSID: %s", ssid);
     wifi_config_t wifi_config = {
     	.ap = {
-    		.ssid = CONFIG_ESP_WIFI_SSID,
-    		.ssid_len = strlen(CONFIG_ESP_WIFI_SSID),

```

Comment:
You removed this value. It should be set based on SSID length.

Reply:
Thanks! I incorrectly read [docs](https://docs.espressif.com/projects/esp-idf/en/v4.4/esp32/api-reference/network/esp_wifi.html#_CPPv4N16wifi_ap_config_t4ssidE).